src/main/java/uk/gov/companieshouse/api/strikeoffobjections/controller/ObjectionController.java

```diff
@@ -118,4 +126,16 @@ public ResponseEntity patchObjection(
             );
         }
     }
+
+    @PostMapping("/{objectionId}/attachments")
+    public ResponseEntity<String> uploadAttachmentToRequest(
```

Comment:
For the method name, we seem to be referring to the requests as Objection in other methods i noticed. So I would either just call it uploadAttachment or uploadAttachmentToObjection

Reply:
Method name changed